### PR TITLE
update FLAdminAPI runner

### PR DESCRIPTION
**Pull request by nvkevlu**
update the FLAdminAPI runner to work better with 2.1, removing some of the features that are no longer relevant or working.

### CODE REVIEW

File: nvflare/fuel/hci/client/fl_admin_api.py [hunk 1]
```diff
@@ -101,6 +101,9 @@ def __init__(
     ):
         """FLAdminAPI serves as foundation for communications to FL server through the AdminAPI.
 
+        Upon initialization, FLAdminAPI will start the overseer agent to get the active server and then try to log in.
+        This happens in a thread, so code that executes after should check that the FLAdminAP is successfully logged in.
```
Comment (YuanTingHsieh) [inline nvflare/fuel/hci/client/fl_admin_api.py:105]: typo FLAdminAP -> FLAdminAPI

**Review by YuanTingHsieh — APPROVED** [review]
LGTM.

File: nvflare/fuel/hci/client/fl_admin_api_runner.py [hunk 2]
```diff
@@ -195,33 +154,5 @@ def run(
             time.sleep(10)  # wait for clients to stop in case they take longer than server to stop
             print("api.check_status(TargetType.CLIENT)")
             api_command_wrapper(self.api.check_status(TargetType.CLIENT))
-            if shutdown_at_end:
-                print("api.shutdown(TargetType.ALL)")
-                api_command_wrapper(self.api.shutdown(TargetType.ALL))
         except RuntimeError as e:
             print(f"There was an exception: {e}")
```
Comment (YuanTingHsieh) [inline nvflare/fuel/hci/client/fl_admin_api_runner.py:158]: so we don't shutdown anymore in the end?

Comment (nvkevlu) [reply nvflare/fuel/hci/client/fl_admin_api_runner.py:158]: shutdown_system still needs implementation work on the server and client side, and shutdown all would not shut down the HA overseer or any other inactive SP

**Pull request merged**